jira_sync/repositories/pagure.py

```diff
@@ -70,13 +70,20 @@ def normalize_issue(self, api_result: dict[str, Any]) -> Issue:
         else:
             status = IssueStatus.closed
 
+        story_points = 0
+
+        for tag in tags:
+            if tag in self.story_points.keys():
+                story_points = self.story_points[tag]
```

Comment:
Maybe …:

```suggestion
                story_points = max(story_points, self.story_points[tag])
```

… so that story points are always set deterministically to the ”maximum value” label. Otherwise, if more than one label is set, it’s kind of random.

Reply:
That is a good suggestion.